docs/detections/prebuilt-rules/rule-details/user-account-exposed-to-kerberoasting.asciidoc

```diff
@@ -0,0 +1,147 @@
+[[user-account-exposed-to-kerberoasting]]
+=== User account exposed to Kerberoasting
+
+Detects when a user account has the servicePrincipalName attribute modified. Attackers can abuse write privileges over a user to configure SPNs so that they can perform Kerberoasting. Administrators can also configure this for legitimate purposes, exposing the account to Kerberoasting.
```

Comment:
```suggestion
Detects when a user account has the servicePrincipalName attribute modified. Attackers can abuse write privileges over a user to configure Service Principle Names (SPNs) so that they can perform Kerberoasting. Administrators can also configure this for legitimate purposes, exposing the account to Kerberoasting.
```